features/install-profiles/src/main/feature/feature.xml

```diff
@@ -54,4 +54,14 @@
         <!--<feature>opendj-embedded</feature>-->
     </feature>
 
+    <feature name="profile-split" version="${project.version}"
+      description="This will install standard application and the Solr Split App."
+      start-level="4">
+        <feature>ddf-boot-features</feature>
+        <feature>catalog-app</feature>
+        <feature>search-ui-app</feature>
+        <feature>split-solr-app</feature>
+        <feature>spatial-app</feature>
+    </feature>
```

Comment:
❓ By arranging the features this way, will the old provider still be lingering around in the service registry?

Reply:
the old provider is part of the `catalog-solr-app` feature.

Reply:
Okay I checked this again and it seems the features are defined well enough that uninstalling a stale component won't be necessary. One profile only starts original Solr. The split profile only starts split Solr. 👍